build/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# TODO: <destdir>/build.log
+# TODO: <destdir>/feeds.conf
+# TODO: print only to stderr
+# TODO: container tar output
+
+# podman run -i --rm alpine:edge sh -c '( apk add git bash wget xz coreutils build-base gcc abuild binutils ncurses-dev gawk bzip2 perl python3 rsync && git clone https://github.com/pktpls/falter-packages.git /root/falter-packages && cd /root/falter-packages/ && git checkout build && build/build.sh master x86_64 out/ ) >&2 && cd /root/falter-packages/out/ && tar -c *' > out.tar
+
+set -ex
+set -o pipefail
+
+function usage() {
+  echo "usage: build/build.sh <branch> <arch> <destination>" >&2
+  exit 1
+}
+
+[ -n "$1" ] && branch="$1" || usage
+[ -n "$2" ] && arch="$2" || usage
+[ -n "$3" ] && dest="$3" || usage
+
+destdir="$dest/snapshots/packages/$arch"
+[ "$branch" == "openwrt-22.03" ] && destdir="$dest/packages-22.03/$arch"
+[ "$branch" == "openwrt-21.02" ] && destdir="$dest/packages-21.02/$arch"
+mkdir -p "$destdir"
+destdir=$(realpath "$destdir")
+
+sdkdir="./tmp/$branch/$arch"
+
+# the grep pipes at the end would otherwise be buffered
+unbuf="stdbuf --output=0 --error=0"
+(
+  mirror="https://downloads.openwrt.org"
+  #mirror="file:///mnt/mirror/downloads.openwrt.org"
```

Comment:
Oh, woould that be the intended way to include custom compiled sdks? Looks interesting. :+1:

Reply:
Yes, just need to recreate the URL structure, e.g. /releases/22.03-SNAPSHOT/packages/x86_64/falter